### PR TITLE
Welcome and SignUp components

### PR DESCRIPTION
**Pull request by RayRedGoose**
### This PR contains the next changes: 

- Add WelcomePage component
- Add styles for WelcomePage component
- Add SignUpForm component
- Add styles for SignUpForm component

### Testings

These changes don't have any tests

### Screenshots

Screen Shot 2020-02-18 at 12 00 18 AM

### CODE REVIEW

File: src/Containers/SignUpForm/SignUpForm.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import './SignUpForm.scss';
+
+interface Props {
+  isLogin: boolean,
+  toggleTab: (login: boolean) => void
+}
+
```
Comment (JamesRexMiller4) [inline src/Containers/SignUpForm/SignUpForm.tsx]: Should we move this into the interfaces file?

File: src/Containers/SignUpForm/SignUpForm.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (JamesRexMiller4) [inline src/Containers/SignUpForm/SignUpForm.scss]: Empty scss file? Do we need to add this?

File: src/css/variables.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+$success-gradient: linear-gradient(130deg, $darken-success-color 0%, $success-color 100%);
+$genius-gradient: linear-gradient(130deg, $darken-genius-color 0%, $genius-color 100%);
+
+// // Mirror Gradients
+// $primary-gradient-mirror: linear-gradient(130deg, $primary-bg-color 0%, $darken-bg-color 100%);
+// $hover-btn-gradient-mirror: linear-gradient(130deg, $regular-hover-color 0%, $darken-regular-hover-color 100%);
+$error-gradient-mirror: linear-gradient(130deg, $error-color 0%, $darken-error-color 100%);
+// $success-gradient-mirror: linear-gradient(130deg, $success-color 0%, $darken-success-color 100%);
+// $genius-gradient-mirror: linear-gradient(130deg, $genius-color 0%, $darken-genius-color 100%);
+
```
Comment (JamesRexMiller4) [inline src/css/variables.scss]: 🤯you are the Gandalf the White of Sass!! Great great work, thank you for tackling this!

File: src/Containers/SignUpForm/SignUpForm.tsx (hunk 1 above)
Comment (RayRedGoose) [reply src/Containers/SignUpForm/SignUpForm.tsx]: No, this is a specific interface for this component only.

File: src/Containers/SignUpForm/SignUpForm.scss (hunk 2 above)
Comment (RayRedGoose) [reply src/Containers/SignUpForm/SignUpForm.scss]: Good catch! I forget to delete it after I replaced styles to welcome page styles.

File: src/css/variables.scss (hunk 3 above)
Comment (RayRedGoose) [reply src/css/variables.scss]: Thanks!

**Comment by RayRedGoose** [comment]
Ok, I've deleted this empty file.

**Pull request merged**